### PR TITLE
Feature/next turn message

### PR DESCRIPTION
**Pull request by bashnia007**

### CODE REVIEW

File: Server.Tests/Characters/KitCarlsonSpecification.cs [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            var game = CreateAndStartGame();
+            var nextPlayer = game.Gameplay.GetNextPlayer();
+            nextPlayer.SetInfo(game.Gameplay, nextPlayer.Role, new KitCarlson());
+
+            var nextPlayerTurnMessage = new NextPlayerTurnMessage(game.Gameplay.PlayerTurn);
+
+            var responseMessage = game.ProcessEvent(nextPlayerTurnMessage);
+            var replayActionMessage = (ReplyActionMessage)responseMessage;
+            var response = (ChooseCardsResponse)replayActionMessage.Response;
+            response.CardsToChoose.Count.Should().Be(3);
```
Comment (IvanovAndrew) [inline Server.Tests/Characters/KitCarlsonSpecification.cs:40]: Just for information: you can write `response.CardsToChoose.Should().HaveCount(3)` instead

File: Bang/Players/Player.cs [hunk 2]
```diff
@@ -265,5 +265,11 @@ public void DropAllCards()
                 DropActiveCard(activeCard);
             }
         }
+
+        public Response EndTurn()
```
Comment (IvanovAndrew) [inline Bang/Players/Player.cs:269]: I'm not sure that method `EndTurn()` should live in `Player`. It looks like `Gameplay` will be a better choice. 
Also I am confused by the name of the method and its body

File: Server.Tests/Characters/KitCarlsonSpecification.cs (hunk 1 above)
Comment (bashnia007) [reply Server.Tests/Characters/KitCarlsonSpecification.cs]: thanks, done

File: Bang/Players/Player.cs (hunk 2 above)
Comment (bashnia007) [reply Bang/Players/Player.cs:269]: done

**Pull request merged**